src/components/collapse/collapse.component.js

```diff
@@ -1,27 +1,27 @@
-import ArrowDownIcon from "@material-ui/icons/KeyboardArrowDown";
-import ArrowUpIcon from "@material-ui/icons/KeyboardArrowUp";
 import clsx from "clsx";
 import React, { useState } from "react";
 import "./style.scss";
 
-const Collapse = ({ collapsed, childArea, linkText }) => {
+const Collapse = ({ collapsed, childArea, linkText, iconExpand, iconCollapse, styleContainer, styleExpansion }) => {
   const [isCollapsed, setIsCollapsed] = useState(collapsed);
 
   return (
     <>
-      <a
-        className="collapse__link"
-        onClick={() => setIsCollapsed(!isCollapsed)}>
-        {linkText}
-        {isCollapsed ? <ArrowUpIcon /> : <ArrowDownIcon />}
-      </a>
-      <a
-        className={clsx("collapse__content",
-          { "collapse__collapsed": isCollapsed },
-          { "collapse__expanded": !isCollapsed })}
-        aria-expanded={isCollapsed}>
-        {childArea}
-      </a>
+      <div className={styleContainer}>
+        <div
+          className={clsx(styleExpansion)}
+          onClick={() => setIsCollapsed(!isCollapsed)}>
+          {linkText}
+          {isCollapsed ? iconExpand : iconCollapse}
```

Comment:
I think that we can take a link to the function from props and render here as component

```suggestion
          {isCollapsed ? <IconExpand /> : <IconCollapse />}
```

Reply:
And we can make an icons by default (arrows up and down)